dts/bindings/input/tsc-keys.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Input driver for STM32 Tocuh Sensing Controller (TSC).
+
+  This node takes a st,stm32-tsc child node and applies
+  filters and calculations to detect an input event
+  Each of the `tsc-keys` child nodes represents a key
+  in the context of the input subsystem, and a group in
+  the context of the TSC controller, for more
+  information see drivers/misc/st,stm32-tsc.yaml
```

Comment:
you can go up to 80 columns here, no need to write the description as a haiku :-)